airbyte-integrations/connectors/source-exchange-rates/build.gradle

```diff
@@ -7,23 +7,3 @@ plugins {
 airbytePython {
```

Comment:
please, add `airbyte-source-acceptance-test` to plugins instead `airbyte-standard-source-test-file`